apps/src/templates/projects/LibraryTable.jsx

```diff
@@ -202,7 +202,6 @@ class LibraryTable extends React.Component {
   actionsFormatter = (_, {rowData}) => {
     return (
       <Button
-        __useDeprecatedTag
```

Comment:
forgive my ignorance here, but when is ok/safe to remove the __useDeprecatedTag?



I remember reverting a similar PR before because it caused some page to not load, so I really want to understand when it is safe to remove. 
https://github.com/code-dot-org/code-dot-org/pull/49608/files

Reply:
I think the problematic buttons in that case were using `href` instead of `onClick` and therefore didn't work when `__useDeprecatedTag` was removed. I also tested the button before and after the change and it still worked in this case :)

Reply:
Can confirm^ the issue was with links disguised as buttons!